backend/scripts/test_litellm_connection.py

```
@@ -0,0 +1,184 @@
+                timeout=30.0,
+            )
+
+            if response.status_code == 200:
+                response_data = response.json()
+                if "choices" in response_data and response_data["choices"]:
+                    content = response_data["choices"][0]["message"]["content"]
+                    print(f"✅ 连接成功！AI回复: {content}")
+                    return True
+                else:
+                    print(f"❌ 响应格式异常: {response_data}")
+                    return False
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)